scripts/release_docs/index.js

```diff
@@ -0,0 +1,66 @@
+import {Octokit} from "octokit";
+import {execSync} from "child_process";
+import {readFile, writeFile} from "fs/promises";
+
+const githubToken = process.argv[2];
+const version = process.argv[3];
+const releaseDate = process.argv[4];
+const amountOfVersions = process.argv[5];
+
+(async function () {
+    const changes = execSync(`conventional-changelog -r ${amountOfVersions}`).toString();
+
+    const lines = changes.split(/\r?\n/);
+    const addedFeatures = lines.filter(line => {
+        return line.startsWith("* feat:")
+    }).map(line => {
+        return line.replace("* feat:", "* Feature: ")
+    }).join("\n");
+
+    const addedFixes = lines.filter(line => {
+        return line.startsWith("* fix:");
+    }).map(line => {
+        return line.replace("* fix:", "* Bugfix: ")
+    }).join("\n");
+
+    const currentChangelog = await readFile("../../CHANGELOG.md");
+    const changeLogLines = currentChangelog.toString().split(/\r?\n/)
+    changeLogLines.shift();
+    changeLogLines.shift();
+    changeLogLines.shift();
+    changeLogLines.shift();
```

Comment:
Could you provide some comment why? To make it clear for future maintenance.

Reply:
Added explanation. THe reason is that I need to replace the first four lines with the generic header.